### DISCUSSION TITLE
Github Education Pack Second Time?

### DISCUSSION THREAD

**Discussion by sandeep-chahal** [community/community#49471] [GitHub Education]
Hello, I previously applied for and received the GitHub Education Pack when I was studying for my Bachelor's degree in India. Now, I have relocated to Canada for further studies and am currently enrolled as a full-time student. I am interested in knowing if I can obtain the GitHub Student Pack again using my new student credentials on the same account.

**Comment by samus-aran** [comment]
Hi @sandeep-chahal 👋 

Yes, you are able to re-apply if you are still currently studying. You can re-verify yourself via the [application page](https://education.github.com/discount_requests/pack_application). 

Hope this helps, if so mark this as an answer to help others 👍
